all.json

```
@@ -49,6 +49,7 @@
     "app-walletconnect.org",
     "app.blockchain-validator.com",
     "app.polkadot.trade",
+    "app.uniswap.org.liquidity-v3.site",
```

```suggestion
    "app.uniswap.org.liquidity-v3.site",
    "liquidity-v3.site",
```

... out of order, CI should take care of ordering